Esther125/CloudDrop
# Feature/yun si/cns 24 roc of redis

## Change
- Add RedisClient that can be reused
- Add one api to test

## Tests
1. Follow the doc (docs/project_setup/redis.md) to install and start redis server
2. call API: http://localhost:3000/api/v1/rooms/redisTest

  If return is
  ```
  {
      "message": "RedisTest"
  }
  ```
  you success
3. wait for 5~10 seconds, open another terminal and run command below

  ```sh
  redis-cli
  
  keys * // to see all key
  ```
  
  the expected output is
  ```
  (empty array)
  ```
  because the key expire in 10 seconds
